module/applications/award.mjs

```diff
@@ -315,7 +315,13 @@ export default class Award extends DialogMixin(FormApplication) {
     }
 
     try {
-      const { currency, xp, party, each } = this.parseAwardCommand(message);
+      let { currency, xp, party, each } = this.parseAwardCommand(message);
+
+      if (xp) {
+        const roll = new Roll(xp);
```

Comment:
I don't think evaluating XP as a roll is necessary to fix this. We are not intending to support variable or rolled XP amounts. All I think is required is to cast the XP value to a number on line 369 (375 in this PR) in `parseAwardCommand`.

```js
else if ( label === "xp" ) xp = Number(amount);
```